media/js/firefox/hello/start.js

```diff
@@ -296,6 +314,10 @@
         // URL query param populated at template level in the view
         var incomingConversation = $main.data('incomingConversation');
 
+        // Get referrer and set Custom Variable. none is okay here.
+        var tourSource = $main.data('tourSource');
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
```

Comment:
As this is the page load for the FTU, "Get Started" was already clicked on a different property (snippet, product page, icon click, etc). We want to attribute every event in the tour to the `Hello FTU Referral`.